image/golem_blender_app/golem_blender_app/commands/utils.py

```diff
@@ -21,53 +24,127 @@ def get_db_connection(work_dir: Path):
 def init_tables(db, subtasks_count: int) -> None:
     with db:
         db.execute(
-            'CREATE TABLE subtask_status(num int, status text, unique_id text)')
-        values = \
-            ((x, SubtaskStatus.PENDING.value) for x in range(subtasks_count))
+            'CREATE TABLE subtasks( '
+            '   num int NOT NULL PRIMARY KEY, '
+            '   id text)')
+        db.execute(
+            'CREATE TABLE history( '
+            '   id text NOT NULL PRIMARY KEY, '
+            '   num int NOT NULL, '
+            '   status text NOT NULL, '
+            '   created DATETIME DEFAULT CURRENT_TIMESTAMP, '
+            '   FOREIGN KEY(num) REFERENCES subtasks(num))')
```

Comment:
To make clearer the distinction between subtasks proper (i.e. assigned jobs) and parts of a task I would recommend the following renaming:
* `subtasks` -> `parts`
     * `id` -> `subtask_id` (actually this field is redundant but we can keep it)
* `history` -> `subtasks`
    * `num` -> `part_num`